### DISCUSSION TITLE
I NEED HELP about the setControlPin

### DISCUSSION THREAD

**Discussion by RC-cn** [HomeSpan/HomeSpan#944] [Q&A]
when i try to set the controlPin by homeSpan.setControlPin(uint8_t (12), triggerType=PushButton::TRIGGER_ON_HIGH); 
I was prompted that 'triggerType' was not declared in this scope 
How can I fix it
![屏幕截图 2024-10-26 173630](https://github.com/user-attachments/assets/f29924f5-4fdd-4ea5-94c0-7703f1e29507)

**Comment by HomeSpan** [comment]
That's not how C or C++ works.  Why are you including the name of the variable (*triggerType*) in your function call?  Should be just:
`homeSpan.setControlPin(12, PushButton::TRIGGER_ON_HIGH);`

**Reply by RC-cn** [reply]
oh,thanks.I understand that . thank you thank you thank you!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
